fuzz/Cargo.toml

```diff
@@ -0,0 +1,36 @@
+[package]
+name = "boa_engine-fuzz"
+version = "0.0.0"
+publish = false
+edition = "2021"
+
+[package.metadata]
+cargo-fuzz = true
+
+[dependencies]
+libfuzzer-sys = "0.4"
+
+[dependencies.boa_ast]
```

Comment:
This can be simplified to `boa_ast = {path = "../boa_ast", features = ["fuzzer-not-safe-for-production"]}`

Reply:
Completed, with similar simplifications for the other crates.